src/player.js

```diff
@@ -1441,6 +1441,10 @@ export default class Player extends FakeEventTarget {
               track.active = false;
             }
           });
+          const currentActiveVideoTrack = newVideoTracks.filter(track => track.active === true);
+          if (!currentActiveVideoTrack.length) {
+            newVideoTracks.at(-1).active = true;
```

Comment:
using at requires updates to the flow config
you can just use length - 1

how do you know which new track to activate ?